src/main/java/digital/slovensko/autogram/core/SigningParameters.java

```diff
@@ -33,13 +33,13 @@ public class SigningParameters {
     private final boolean checkPDFACompliance;
     private final int visualizationWidth;
 
+    private Exception transformationException = null;
```

Comment:
Toto sa mi uplne nepozdava. Asi by to chcelo nejaky getTransformedData alebo nieco take, co pripadne hodi/vrati error. Lebo tuto to teraz takto skaredo hackujeme.

Reply:
ja som si konecne otvoril #186 a vramci toho sa snazim to upratat, je to ale kusok rozsiahlejsie - je to ale este in progress bohuzial.
https://github.com/slovensko-digital/autogram/compare/main...sabomichal:autogram:feature/186_checked_exceptions.

Reply:
tam ak dobre vidim, tak toto ignorovanie exceptionu je problem - potrebujeme vediet rozoznat ked nieco nema transformaciu a ked malo transformaciu a ta failla (a tam by sme chceli ukazat exception aby bolo jasne ci ide o problem v xslt alebo na nasej strane) to je vlastne to co som sa snazil urobit cez `transformationException` co mi je jasne, ze nie je idealny pattern.

https://github.com/slovensko-digital/autogram/compare/main...sabomichal:autogram:feature/186_checked_exceptions#diff-04c631c332ea39327cf1287bc70ed40d9724d9a2a4a47c0d9f298c8a65cdea02R72

Moja otazka je ze ci je OK ze by gettery hadzali exceptions? Lebo mi pride ako najjednoduchsie riesenie ak to je ok.

Reply:
Ak chces explicitne vediet, ze transformacia sa nepodarila nacitat kvoli chybe, a nielen ju v tomto pripade povazovat za neznamu (null), tak ano, exception je samozrejme ok.